extras/AFC_functions.py

```diff
@@ -206,6 +222,16 @@ def check_homed(self):
             if self.afc.auto_home:
                 self.afc.gcode.run_script_from_command("G28")
                 self.afc.toolhead.wait_moves()
+                if self.afc.auto_level:
+                    if not self.afc.auto_level_macro:
+                        self.afc.error.AFC_error("Auto level is enabled but no auto level macro defined", False, level=2)
+                        return False
+                    if self.check_macro_present(self.afc.auto_level_macro):
+                        self.afc.gcode.run_script_from_command(self.afc.auto_level_macro)
+                        self.afc.toolhead.wait_moves()
+                    else:
+                        self.afc.error.AFC_error("Auto level macro defined, but not found in printer configuration.", False, level=2)
+                        return False
```

Comment:
So instead of having two variables we could just keep `auto_level_macro` and then the logic would change to:
```suggestion
                if self.afc.auto_level_macro is not None:
                    if self.check_macro_present(self.afc.auto_level_macro):
                        self.afc.gcode.run_script_from_command(self.afc.auto_level_macro)
                        self.afc.toolhead.wait_moves()
                    else:
                        self.afc.error.AFC_error("Auto level macro defined, but not found in printer configuration.", False, level=2)
                        return False
```
Or you can keep it as is, just something I noticed that could be done